docs/contributing.md

```diff
@@ -0,0 +1,74 @@
+# Contributing
+
+We very much welcome ideas, questions and relevant contributions from anyone
+with an interest in improving the adoption and usage of free-threaded CPython.
+
+## Contributing to this site
+
+Contributions can be made through issues on and pull requests to the
+[free-threaded-compatibility](https://github.com/Quansight-Labs/free-threaded-compatibility)
+repository. Ways to contribute include:
+
+- Improvements to the documentation, from new content to copy-editing for clarity.
+- Updates to the [status tracker](https://py-free-threading.github.io/tracking/).
+- Adding links to relevant examples, blog posts and other content.
+- For particularly exciting use cases or benchmarks, including them directly
+    into the site instead of only linking out to them.
+
+## Contributing to the adoption of free-threading
+
+There's a ton of work to do here - and a ton of fun and performance to be had!
+This is perhaps the single most impactful and exciting change to CPython since
+the Python 2 to Python 3 transition. It's going to take the community multiple
+years to complete the transition to free-threaded CPython, and your help will
+be valuable. Here are some ways to contribute today:
+
+1. Start experimenting! If the packages you rely on for some task or application,
+    try running it with a free-threaded interpreter. And consider whether it can
+    benefit performance or functionality-wise from Python-level threading.
+1. Implement support for free-threading in packages that are still lacking
```

Comment:
```suggestion
2. Implement support for free-threading in packages that are still lacking
```

I *think* it doesn't actually matter in markdown though

Reply:
This is pre-commit messing up - basically all the changes in the commit it added are making things worse. If I commit this suggestion, it'll just change it back again.